packages/react-strict-dom/postcss/plugin.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ */
+
+const styleXPlugin = require('@stylexjs/postcss-plugin');
+
+const plugin = ({
+  cwd = process.cwd(),
+  // By default reuses the Babel configuration from the project root.
+  // Use `babelrc: false` to disable this behavior.
+  babelConfig = {},
+  include,
+  exclude
+}) => {
+  include = [
+    // Include the React Strict DOM package's source files by default
+    require.resolve('react-strict-dom'),
```

Comment:
I think we need to include the runtime export too

Reply:
Added in this commit
https://github.com/facebook/react-strict-dom/pull/308/commits/e760f8c94d4bfe39a8f96a237caf4179313c0c1d